BE/src/main/java/yanse/airbnb/auth/LoginService.java

```diff
@@ -0,0 +1,28 @@
+package yanse.airbnb.auth;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import yanse.airbnb.auth.dto.AccessToken;
+import yanse.airbnb.auth.github.GithubUser;
+import yanse.airbnb.domain.member.Members;
+import yanse.airbnb.domain.member.repository.MembersRepository;
+
+@RequiredArgsConstructor
+@Service
+public class LoginService {
+
+	private final MembersRepository membersRepository;
+
+	//TODO:예외처리
+	public Members findLoginUser(Members members) {
+		if (!isDuplicate(members)) {
+			membersRepository.save(members);
+			return members;
+		}
+		return members;
```

Comment:
굳이 if 문 내부를 한번 더 생각하는 부정문으로 처리하기보다는, 아래처럼 가는편이 더 쉽게 읽힐 것 같아요.

```suggestion
	public Members findLoginUser(Members members) {
		if (isDuplicate(members)) {
			return members;
		}
		return membersRepository.save(members);
	}
```